flask-backend/requirements.txt

```diff
@@ -1,23 +1,228 @@
+alabaster==0.7.12
```

Comment:
They're a lot of dependencies, not all are required just to introduce msg extraction.
Remove redundant imports.

Reply:
@shivanshu1333 updated!

Reply:
It looks like you reverted all the changes, are you sure no additional dependency is required here?

Reply:
Yes sir! No other dependency needed now. These dependencies came from the last PR #173  when setting up the flask. @sachinsom93 added the required dependencies in #176

Reply:
Agreed with @swapnalshahil, sir @shivanshu1333 , actually I forgot to update requirements.txt file in #173.But, I have updated requirements.txt file in #176 .

Reply:
Alright!